examples/hcp-vm-demo/main.tf

```
@@ -70,6 +70,11 @@ resource "hcp_consul_cluster_root_token" "token" {
   cluster_id = hcp_consul_cluster.main.id
 }
 
+resource "tls_private_key" "ssh" {
+  algorithm = "RSA"
+  rsa_bits  = 4096
```

Probably best to use Ed25519:
```suggestion
  algorithm = "Ed25519"
```